README.md

```diff
@@ -208,11 +208,12 @@ tenderly actions deploy
 
 Make sure you configure the secrets in Tenderly:
 
-- `SLACK_WEBHOOK_URL`: Slack Webhook
-- `NOTIFICATIONS_ENABLED`: (default `true`) Set to `false` to disable Slack notifications
 - `NODE_URL_${network}`: RPC Node URL
 - `NODE_USER_${network}`: (optional) RPC Node user name for basic auth.
 - `NODE_PASSWORD_${network}`: (optional) RPC Node password name for basic auth.
+- `NOTIFICATIONS_ENABLED`: (default `true`) Set to `false` to disable Slack notifications
+- `SLACK_WEBHOOK_URL`: Slack Webhook (required only if notifications are enabled)
+- `SENTRY_DSN`: ()optional Sentry DSN code. If present, it will enable Sentry notifications
```

Comment:
What it supposed to have anything within the parenthesis?

Reply:
ahaha, the optional 🙈